### ISSUE TITLE
updater stuck

### ISSUE THREAD

**Issue by Limdel** [Nefelim4ag/League-Of-Legends#15]
It's probably not related to script, but I'm not sure and I can't fix it myself.
After last update, beta client launches with updater stuck at 0.0 MBs / 0.0 MBs. Same thing happens to some Windows users and there is solution for it, which sadly didn't help me. I have dualboot and and under Windows everything works, problem appears only under linux. Also, I tried doing full client repair, which didn't help too.

Logs: http://hastebin.com/zamukawufe.coffeescript

PLZ HALP

**Comment by Nefelim4ag** [comment]
@Limdel, about your log:
Why you did not use general launcher for update client? In logs i see that you only try to use new beta client.

Also:
1. Try use default setup i.e. wine, not staging + default deps
2. Try switch wine arch, i.e. set it to x64/x32

**Comment by Limdel** [comment]
> Why you did not use general launcher for update client?
Client is already updated. Did it under wongblows.

> Try use default setup i.e. wine, not staging + default deps (or just default deps)
>Try switch wine arch, i.e. set it to x64/x32
Tried everything mentioned.

**Comment by Limdel** [comment]
Okay. I finally found what was the problem. Patch that was suppose to fix bug splat problem related to launching wrong LeagueClient.exe by cutting only last result using tail command actually borked it for me. It tries to launch exe form "installer" folder instead of folder with version number, which doesn't work, but when I manually replaced path to folder with number I managed to launch it.
I don't know why it doesn't work for me, and does for others. Any explanation?
